js/inject.js

```diff
@@ -60,20 +60,20 @@
 	let   lng = chrome.i18n.getUILanguage();
 	const uLng = localStorage.getItem('user-language');
 
-	// wir brauchen nur den ersten Teil
+	// we only need the first part
 	if (lng.indexOf('-') > 0) {
 		lng = lng.split('-')[0];
 	}
 
-	// gibt es eine Übersetzung?
+	// is there a translation?
 	if (Languages.PossibleLanguages[lng] === undefined) {
 		lng = 'en';
 	}
 
 	if (uLng !== null){
 		lng = uLng;
 	} else {
-		// damit man die sprache auslesen kann ohne diese über die einstellungen einmal ändern zu müssen
+		// so that the language can be read out without having to change it once via the settings
```

Comment:
change to:
to read the language without the condition of changing it once in the settings